src/g3log/g3log.hpp

```diff
@@ -145,17 +145,17 @@ namespace g3 {
 
 
 // LOG(level) is the API for the stream log
-#define LOG(level) if(!g3::logLevel(level)) {} else INTERNAL_LOG_MESSAGE(level).stream()
+#define LOG(level) if (!g3::logLevel(level)) {} else INTERNAL_LOG_MESSAGE(level).stream()
```

Comment:
Format change with my  “space” change not yet removed

Reply:
This is the only place you didn't put a space after an `if`.
Are you sure that you want to keep it _inconsistent_ with everywhere else?

Reply:
You are right. For this file. Leave this as is. Thanks for calling it out